src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -1,5 +1,43 @@
 package org.apache.maven.plugins.javadoc;
 
+import static org.apache.maven.plugins.javadoc.JavadocUtil.isEmpty;

```

Comment:
This is a not a well known class so static imports should probably be avoided.

Reply:
Hi Elliotte 
Sorry I messed up this commit, because my IDE reorganized the whole import section of this file. I reverted that in my last commit. I could squash the commits to get a clearer history if that is recommended. However I have not introduced this static import in this commit here I only moved it around. 
Do I still have to refactor this static import? This will introduce some additional changes to this PR that are not relevant to the issue I try to fix.

Reply:
I missed that the imports were reorganized. If this is existing code, no need to fix it here. I don't know if maven follows any conventions for import ordering, but absent that it's best not to reorder imports to avoid churn.